src/main/java/com/example/pokebattlez/controller/ChatController.java

```diff
@@ -88,17 +91,25 @@ public void challenge(SimpMessageHeaderAccessor sha, ChallengeReceive chr) {
     }
 
     @MessageMapping("/chat/challenge/accept")
-    public void acceptChallenge(SimpMessageHeaderAccessor sha, @RequestParam("accept") boolean accept, @RequestParam("from") Long from) {
+    public void acceptChallenge(SimpMessageHeaderAccessor sha, ChallengeResponse cr) {
```

Comment:
I understand the **sha** is a way to obfuscate the user identity.

Reply:
For the moment it's the only reliable way of identifying connections that we found, it's gonna get replaced with JWT when we implement that.